test/index.spec.js

```diff
@@ -109,11 +114,13 @@ describe('DelegatedContentRouting', function () {
     const cid = new CID('QmS4ustL54uo8FzR9455qaxZwuMiUhyvMcX9Ba8nUH4uVv')
 
     before('register providers', async () => {
-      await bootstrapNode.api.dht.provide(cid)
-      await selfNode.api.dht.provide(cid)
+      await drain(bootstrapNode.api.dht.provide(cid))
+      await drain(selfNode.api.dht.provide(cid))
     })
 
-    it('should be able to find providers through the delegate node', async () => {
+    it('should be able to find providers through the delegate node', async function () {
+      this.timeout(60000)
```

Comment:
Is this timeout necessary? Running this locally and looking at CI, this test finishes in milliseconds.

Reply:
Perhaps not - there was definitely a point in time where this helped but it could have been something else - if it's fast in CI I'll remove it.